src/components/Expander/Expander.jsx

```diff
@@ -109,25 +119,28 @@ const Expander = createClass({
 			className,
 			isExpanded,
 			style,
+			kind,
 			...passThroughs,
 		} = this.props;
 
 		const labelChildProp = _.first(_.map(findTypes(this.props, Expander.Label), 'props'));
+		const withEmphasis = kind === 'emphasis';
 
 		return (
 			<div
 				{...omitProps(passThroughs, Expander)}
 				className={cx('&', {
 					'&-is-expanded': isExpanded,
+					'&-with-emphasis': withEmphasis,
 				}, className)}
 				style={style}
 			>
 				<header className={cx('&-header')} onClick={this.handleToggle}>
-					<span className={cx('&-icon')}>
+					{!withEmphasis ? (<span className={cx('&-icon')}>
```

Comment:
Since this is used in two places, make a component in this same file like this:

```javascript
const ChevronButton = createClass({
	propTypes: {
		isExpanded: bool,
	},

	render() {
		return (
			<span className={cx('&-icon')}>
				<ChevronIcon
					direction={this.props.isExpanded ? 'up' : 'down'}
				/>
			</span>
		);
	},
});
```

and then use it in both places like so:

```javascript
					{!withEmphasis ? <ChevronButton isExpanded={isExpanded} /> : null}
```